hicexplorer/hicBuildMatrix.py

```diff
@@ -5,7 +5,15 @@
 import time
 from os import unlink
 import os
+import shutil
 import pysam
+from six.moves import xrange
```

Comment:
I am not sure how the six package works (haven't use it before), but does this means that we need to add a new dependency to setup.py?